test/functional/apps/dashboard/dashboard_state.js

```diff
@@ -278,13 +280,15 @@ export default function ({ getService, getPageObjects }) {
           await retry.try(async () => {
             const pieSliceStyle = await pieChart.getPieSliceStyle('80,000');
             // The default green color that was stored with the visualization before any dashboard overrides.
```

Comment:
```suggestion
            // The default color that was stored with the visualization before any dashboard overrides.
```
No guarantee it's a green